src/frontend/src/catalog/root_catalog.rs

```diff
@@ -94,6 +95,8 @@ pub struct Catalog {
     db_name_by_id: HashMap<DatabaseId, String>,
     /// all table catalogs in the cluster identified by universal unique table id.
     table_by_id: HashMap<TableId, TableCatalog>,
+    connection_by_id: HashMap<ConnectionId, ConnectionCatalog>,
+    connection_id_by_name: HashMap<String, ConnectionId>,
```

Comment:
Actually, these fields should be put in the `SchemaCatalog`. Any database objects should be in the hierarchy of "database -> schema".

Reply:
yes we will fix it in the next pr :)